### PR TITLE
lp_solve: mark as broken

### PR DESCRIPTION
**Pull request by joachifm**
See https://hydra.nixos.org/build/32393768/log/raw

### CODE REVIEW

File: pkgs/applications/science/math/lp_solve/default.nix [hunk 1]
```diff
@@ -37,6 +37,7 @@ stdenv.mkDerivation rec {
     license     = licenses.gpl2Plus;
     maintainers = with maintainers; [ smironov ];
     platforms   = platforms.unix;
+    broken = false;
```
Comment (zimbatm) [inline pkgs/applications/science/math/lp_solve/default.nix]: I think you meant `broken = true`. Can you also align the `=` with the rest for concistency's sake ?

Comment (joachifm) [inline pkgs/applications/science/math/lp_solve/default.nix]: Hah, thanks for catching that

**Comment by zimbatm** [comment]
This might be related to the glibc upgrade. /cc @vcunat to see if there is any quick fix possible.

**Comment by joachifm** [comment]
My working hypothesis was that it re-defines some things due to failing to detect that it shouldn't, but I simply could not find the motivation to figure out if that's actually the case.

**Comment by zimbatm** [comment]
Alright, we can always mark it as unbroken if a fix is submitted.

**Comment by vcunat** [comment]
Yes, this was triggered by glibc upgrade and tried to fix this when the upgrade was new, but I didn't get to understand this whole `FALSE` business and the package didn't seem important enough (to me) to invest more time.

**Pull request merged**